components/brave_rewards/browser/publisher_info.h

```diff
@@ -0,0 +1,37 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
```

Comment:
We have `ContentSite` as well which acts as holder for `ledger::PublisherInfo`. If we switch to this one, we need to remove and migrate `ContentSite` so that we have only one. `ContentSite` is confusing anyway

Reply:
Sure, removed `ContentSite` and switched to use `PublisherInfo`.